packages/selleo-design-core/src/Paragraph.tsx

```diff
@@ -0,0 +1,35 @@
+import cx from "classnames";
+import { h } from "preact";
+
+type ParagraphProps = {
+  size: "large" | "medium" | "small";
+  weight: "regular" | "bold";
+};
+
+type Variants = `${NonNullable<ParagraphProps["size"]>}/${NonNullable<
+  ParagraphProps["weight"]
+>}`;
+
+interface Props extends Omit<ParagraphProps, "size" | "weight"> {
+  variant: Exclude<Variants, "small/bold">;
+}
+
+export function Paragraph({
+  variant = "large/regular",
+}: React.PropsWithChildren<Props>) {
+  const paragraphClassname = cx(
+    "dark:text-white font-sans tracking-normal text-neutral-400",
+    {
+      "text-base font-normal leading-[175%]": variant === "large/regular",
+      "text-base font-bold leading-[175%]": variant === "large/bold",
+      "text-sm font-normal leading-[175%]": variant === "medium/regular",
+      "text-sm font-bold leading-[175%]": variant === "medium/bold",
+      "text-xs font-normal leading-[150%]": variant === "small/regular",
+    }
+  );
```

Comment:
variant and size should be handled separately - check @Matgorn's PR
https://github.com/Selleo/design-system/pull/4/files